From b9a65bf3ba5628cfe8cfd2d10ce2dcf11a606775 Mon Sep 17 00:00:00 2001
From: Dongxiao Xu <dongxiao.xu@intel.com>
Date: Thu, 14 Jul 2011 15:40:36 +0800
Subject: [PATCH] alsa-tools: Fix recipe build error.

Add parameters to autoreconf to support cross compile.
Remove some sub-components which needs further recipe support.

Signed-off-by: Dongxiao Xu <dongxiao.xu@intel.com>

Upstream-Status: Inappropriate [configuration]

---
 Makefile          | 4 ++--
 ld10k1/gitcompile | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index c32bf25..1119372 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
 VERSION = 1.1.6
 TOP = .
-SUBDIRS = as10k1 envy24control hdsploader hdspconf hdspmixer \
+SUBDIRS = as10k1 envy24control \
 	  mixartloader pcxhrloader rmedigicontrol sb16_csp seq sscape_ctl \
-	  us428control usx2yloader vxloader echomixer ld10k1 qlo10k1 \
+	  us428control usx2yloader vxloader echomixer \
 	  hwmixvolume hdajackretask hda-verb hdajacksensetest
 
 all:
diff --git a/ld10k1/gitcompile b/ld10k1/gitcompile
index 99429ac..20005d9 100755
--- a/ld10k1/gitcompile
+++ b/ld10k1/gitcompile
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-autoreconf -fi || exit 1
+autoreconf $ACLOCAL_FLAGS -fi || exit 1
 export CFLAGS='-O2 -Wall -pipe -g'
 echo "CFLAGS=$CFLAGS"
 echo "./configure $@"
